### PR TITLE
Feat/url uploading

### PR DESCRIPTION
**Pull request by Molnes**
Add functionality to allow uploading to the bucket from an url

### CODE REVIEW

File: internal/web_server/web/handlers/api/admin_api.go [hunk 1]
```diff
@@ -127,6 +127,16 @@ func (aah *AdminApiHandler) editQuizImage(c echo.Context) error {
...
 	}
 
+	imageName, err := aah.uploadImageFromURL(c, *imageURL)
+	if err != nil {
+		return utils.Render(c, http.StatusBadRequest, components.ErrorText(errorImageElementID, "Kunne ikke laste opp bilde"))
+	}
+
+	imageURL, err = url.Parse(aah.sharedData.Bucket.EndpointURL().String() + "/images/" + imageName)
+	if err != nil {
+		return utils.Render(c, http.StatusBadRequest, components.ErrorText(errorImageElementID, "Kunne ikke laste opp bilde"))
```
Comment (z-rayc) [inline internal/web_server/web/handlers/api/admin_api.go:137]: You changed below `Kunne ikke laste opp bilde` to `bildet`. Do so here as well.

File: internal/web_server/web/handlers/api/admin_api.go [hunk 2]
```diff
@@ -127,6 +127,16 @@ func (aah *AdminApiHandler) editQuizImage(c echo.Context) error {
 		return utils.Render(c, http.StatusBadRequest, components.ErrorText(errorImageElementID, "Ugyldig bilde URL"))
 	}
 
+	imageName, err := aah.uploadImageFromURL(c, *imageURL)
```
Comment (z-rayc) [inline internal/web_server/web/handlers/api/admin_api.go:130]: Is there any security against uploading the image if the URL is already to the bucket?

File: internal/web_server/web/handlers/api/admin_api.go [hunk 3]
```diff
@@ -166,7 +176,7 @@ func (aah *AdminApiHandler) uploadQuizImage(c echo.Context) error {
 	imageAsURL, err := url.Parse(imageURL)
 	if err != nil {
 		log.Println(err)
-		return utils.Render(c, http.StatusBadRequest, components.ErrorText(errorImageElementID, "Kunne ikke laste opp bilde"))
+		return utils.Render(c, http.StatusBadRequest, components.ErrorText(errorImageElementID, "Kunne ikke laste opp fullføre bildeopplastingen"))
```
Comment (z-rayc) [inline internal/web_server/web/handlers/api/admin_api.go:179]: `Kunne ikke laste opp fullføre bildeopplastingen` is a nonsensical sentence. Fix.

File: internal/web_server/web/handlers/api/admin_api.go (hunk 2 above)
Comment (Molnes) [reply internal/web_server/web/handlers/api/admin_api.go:130]: as best as i can understand, the file will just be replaced, tho that is unlikely given the nature of uuidv4

**Review by z-rayc — APPROVED** [review]

**Pull request merged**